cmd/gateway/commands.go

```diff
@@ -116,10 +116,10 @@ func createControlPlaneCommand() *cobra.Command {
 	}
 }
 
-func createProvisionerCommand() *cobra.Command {
+func createProvisionerModeCommand() *cobra.Command {
 	return &cobra.Command{
-		Use:    "provisioner",
-		Short:  "Start the provisioner",
+		Use:    "provisioner-mode",
+		Short:  "Provision an NGINX Gateway Deployment per Gateway resource configured in the static mode",
```

Comment:
Do we need the "configured in the static mode" part of the sentence? It's a bit confusing to me.

Reply:
my idea was to clarify that it will provision NKG in the static mode, which is the other command. I wonder if you have a suggestion here

Reply:
maybe: 
```suggestion
		Short:  "Provision a static-mode NGINX Gateway Deployment per Gateway resource",
```